packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -51,16 +51,10 @@ describe(`Tests for webpack plugin`, function() {
     });
   });
 
-  afterEach(function() {
-    this.timeout(10 * 1000);
-
-    return fsExtra.remove(tmpDirectory);
-  });
-
   // Kill the web server once all tests are complete.
   after(function() {
     this.timeout(10 * 1000);
-
+    fsExtra.remove(tmpDirectory);
```

Comment:
Use `removeSync()` here (or return a promise chain that includes the `remove()` promise).